datafusion/core/tests/user_defined/user_defined_scalar_functions.rs

```diff
@@ -43,7 +43,7 @@ async fn csv_query_custom_udf_with_cast() -> Result<()> {
         "+------------------------------------------+",
         "| AVG(custom_sqrt(aggregate_test_100.c11)) |",
         "+------------------------------------------+",
-        "| 0.6584408483418833                       |",
+        "| 0.6584408483418835                       |",
```

Comment:
this differs in the last decimal place and thus I think is related to floating point stability and thus this change is fine